lib/kafka/produce_operation.rb

```diff
@@ -1,4 +1,25 @@
 module Kafka
+  # A produce operation attempts to send all messages in a buffer to the Kafka cluster.
+  # Since topics and partitions are spread among all brokers in a cluster, this usually
+  # involves sending requests to several or all of the brokers.
+  #
+  # ## Instrumentation
+  #
+  # When executing the operation, a `append_message_set.kafka` notification will be
+  # emitted for each message set that was successfully appended to a topic partition.
```

Comment:
All the time I’m reading “message set” as if a message has been set. Would it be possible to write `message_set` instead?